workloads/server.hbs.md

```diff
@@ -105,3 +105,285 @@ The following values are valid within the `ports` argument:
 | `port` | The port on which the application is exposed to the rest of the cluster |
 | `containerPort` | The port on which the application listens for requests. Defaults to `port` if not set. |
 | `name` | A human-readable name for the port. Defaults to `port` if not set. |
+
+
+## <a id="params"></a> Securing `server` workloads
+
+### Manual configuration
+For http `server` workloads you can expose them securely creating an `Ingress` resource. Taking the `spring-sensors-consumer-web`
+workload as an example, create the following `Ingress`:
+
+```
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: spring-sensors-consumer-web
+  namespace: <YOUR-DEVELOPER-NAMESPACE>
+  annotations:
+    cert-manager.io/cluster-issuer: tap-ingress-selfsigned
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: contour
+    kubernetes.io/tls-acme: "true"
+spec:
+  tls:
+    - secretName: spring-sensors-consumer-web
+      hosts:
+        - "spring-sensors-consumer-web.<INGRESS-DOMAIN>"
+  rules:
+    - host: "spring-sensors-consumer-web.<INGRESS-DOMAIN>"
+      http:
+        paths:
+          - pathType: Prefix
+            path: /
+            backend:
+              service:
+                name: spring-sensors-consumer-web
+                port:
+                  number: 8080
+
+```
+
+Replace the `<YOUR-DEVELOPER-NAMESPACE>` with your developer namespace and `<INGRESS-DOMAIN>` with the domain name defined
+in `tap-values.yaml` during the installation. Also set the annotation `cert-manager.io/cluster-issuer` to the `ingress_issuer`
+set during installation or leave as `tap-ingress-selfsigned` to use the default one. Make sure to update the port exposed by
+your `Service` resource, in the previous snippet it is set to `8080`. 
+
+Your `server` workload can be accessed with https like so:
+```console
+curl -k https://spring-sensors-consumer-web.<INGRESS-DOMAIN>
+```
+
+
+### Creating a new workload type that will secure _server_ workloads by default
+
+>**Note** Make sure you delete the `Ingress` resource previously created.
+
+TAP allows to create new workload types. Start by taking the existing `server-template` `ClusterConfigTemplate` and modify
+it to add an `Ingress` resource when this new type of workload is created.
+
+>**Note** This procedure assumes you have `yq` cli installed in your computer. 
+
+1. Save the existing `server-template` in a local file:
+```console
+kubectl get ClusterConfigTemplate server-template -oyaml > secure-server-template.yaml
+```
+2. Extract `.spec.ytt` field from this file and create another file
+```console
+yq '.spec.ytt' secure-server-template.yaml > spec-ytt.yaml
```

Comment:
Can you check if your yq version works with yq eval instead? mine doesn't work only specifying the yaml path without using eval. e.g yq eval '.spec.ytt' secure-server-template.yaml > spec-ytt.yaml

Reply:
I found this: https://mikefarah.gitbook.io/yq/

```
Notice for v4.x versions prior to 4.18.1
Since 4.18.1, yq's 'eval/e' command is the default command and no longers needs to be specified.
Older versions will still need to specify 'eval/e'.
Similarly, '-' is no longer required as a filename to read from STDIN (unless reading from multiple files).
```

this is my version:
```
yq (https://github.com/mikefarah/yq/) version v4.30.5
```

it works with `eval` as well

Reply:
I tried with the yq we use on CI. So, my suggestion is to add "eval" since it works for both, just in case.